### PR TITLE
GP11 : Key derivation function fix

### PR DESCRIPTION
**Pull request by cedric-chaumont-st-dev**
Signed-off-by: Cedric Chaumont <cedric.chaumont@st.com>

### CODE REVIEW

File: lib/libutee/tee_api_operations.c [hunk 1]
```diff
@@ -1300,7 +1300,7 @@ void TEE_DeriveKey(TEE_OperationHandle operation,
 
 	if (operation == TEE_HANDLE_NULL || derivedKey == 0)
 		TEE_Panic(0);
-	if (paramCount != 0 && params == NULL)
+	if (params == NULL && paramCount != 0)
```
Comment (jbech-linaro) [inline lib/libutee/tee_api_operations.c]: What is the magic here?

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`
(Even though I don't get one of the changes ...)

File: lib/libutee/tee_api_operations.c (hunk 1 above)
Comment (cedric-chaumont-st-dev) [inline lib/libutee/tee_api_operations.c]: Only to be consistent with PR #379. I can revert that if you prefer.

**Comment by cedric-chaumont-st-dev** [comment]
Rebased.

**Comment by cedric-chaumont-st-dev** [comment]
Rebased. Tested (STM, ARM Juno).
28701 subtests of which 0 failed
687 test cases of which 0 failed
0 test case was skipped
TEE test application done!

**Pull request merged**